validation/validator.go

```diff
@@ -21,16 +21,37 @@ func init() {
 
 type Validator struct {
 	instance *validate.Validation
+	data     validate.DataFace
 }
 
-func NewValidator(instance *validate.Validation) *Validator {
+func NewValidator(instance *validate.Validation, data validate.DataFace) *Validator {
 	instance.Validate()
 
-	return &Validator{instance: instance}
+	return &Validator{instance: instance, data: data}
 }
 
 func (v *Validator) Bind(ptr any) error {
-	data := v.instance.SafeData()
+	// Don't bind if there are errors
+	if v.Errors() != nil {
+		return nil
+	}
+
+	var data any
+	if formData, ok := v.data.(*validate.FormData); ok {
+		values := make(map[string]any)
+		for key, value := range v.data.Src().(url.Values) {
+			values[key] = value[0]
```

Comment:
Only support one value in the form case, this is the default behavior of gookit/validate for now.